tests/cli/test_start_agent.py

```diff
@@ -111,6 +113,7 @@ async def test_sigint_sends_sigterm_then_sigkill(self, agent_process):
         agent_process.send_signal(signal.SIGINT)
         await anyio.sleep(0.01)  # some time needed for the recursive signal handler
         agent_process.send_signal(signal.SIGINT)
+        await anyio.sleep(0.1)  # reduce flakiness
```

Comment:
Is there another way we can wait for the signal here? This is going to be brittle with CI machines which are really slow randomly.

Reply:
in that case, potentially a simple increase of the SHUTDOWN_TIMEOUT vatiable could solve flakiness?

Reply:
oh wow that really did the trick. we're green :)